src/app.module.ts

```diff
@@ -160,6 +161,10 @@ export class AppModule implements NestModule {
           provide: APP_INTERCEPTOR,
           useClass: CacheControlInterceptor,
         },
+        {
+          provide: APP_INTERCEPTOR,
```

Comment:
This might need be localised to datasources, otherwise different calls to datasources within a mapping might still make separate requests.

Reply:
I've just experimented some more and this is indeed the case.